frontend/src/pages/Courses.jsx

```
@@ -58,15 +58,15 @@ const Courses = () => {
 
   return (
     <div className={styles.container}>
-      <div className={styles.card} onClick={() => setShowForm(true)}>
+      <button className={styles.card} onClick={() => setShowForm(true)}>
         <div 
           className={styles.icon}
         >
           <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" className="size-6">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)